transform/surveys/023.0102.json

```diff
@@ -27,19 +99,39 @@
       ]
     },
     {
-      "title": "4. Retail Turnover",
+      "title": "3. Commodoties - Retail Turnover",
       "meta": "(£)",
       "questions": [
         {
+          "text": "Food",
```

Comment:
I checked with the EQ dev page and all the extra questions should be gone I think